pages/documents/MessagingChannels/MobileAppMessagingSDKforiOS/ReleaseNotes/regular/5.2.0.md

```diff
@@ -0,0 +1,55 @@
+### Version 5.2.0
+#### iOS Messaging SDK
+
+**Release Date**: May 27, 2020
+
+##### Environmental Requirements
+The iOS Mobile Messaging SDK version 5.2.0 is supported on iOS versions 11 through 13. It is available on Swift version 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15), and Swift version 5.2.4 (swiftlang-1103.0.32.9 clang-1103.0.32.53)
```

Comment:
Do we also need to specify Xcode version ?

Reply:
We don't specify the Xcode version here.